src/web/core/src/main/java/org/geoserver/web/css/geoserver.css

```diff
@@ -1448,4 +1448,13 @@ li.select2-results__option {
 
 .fullscreen #cm_editor_heigth {
    enabled: false;
-}
\ No newline at end of file
+}
+
+/* --- */
+.default-margin {
```

Comment:
If it's a custom class it's not really a default margin... or not?

Reply:
It is my proposal for a new "default-margin" style making UI components looking a bit better thanks to visual separation from others on the form. 
Without any margin, text fields are less readable because they're too close.

Reply:
Styling should be consistent across the whole user interface. Can you show us the results in both cases, with and without the css changes?

Reply:
Sure. [Here](https://drive.google.com/open?id=1CXLfIk00nNrcJQJasNPLOkmX0jhM78ip) is our UI with margin (and second style, for width) enabled.
And [here](https://drive.google.com/open?id=1VB3Zdm69bB7rcoxVJaO__wUA_4Kt-XKv) with both disabled